8.x/core-concepts/validation.md

```diff
@@ -97,9 +119,7 @@ class Kernel extends HttpKernel{
 }
 ```
 
-:::tip
-The requests that come from the client to the server must pass through some layers: ((router -> BeforeMiddleware -> controller method -> AfterMiddleware -> response)). The curious reader can refer to the Laravel documentation in order to study more about the Middlewares.*<sup>[2](#2)</sup>*
-:::
+**NOTE:** The curious reader can refer to the Laravel documentation in order to study more about the Middlewares.*<sup>[3](#3)</sup>*
 
 The `RuleMiddleware` class has a method called `handle()` which receives two input parameters: `$request` and `Closure $next`:
 
```

Comment:
Please write the file path either in the comment above the codes or in your text above the code block.

Reply:
@reyhane-rangamiz : Done.